org.adempiere.base/src/org/compiere/model/MAllocationHdr.java

```diff
@@ -947,17 +947,18 @@ private boolean updateBP()
 		} // for all lines
 		return true;
 	}	//	updateBP
-	
+
 	/**
-	 * 	Document Status is Complete or Closed
-	 *	@return true if CO, CL or RE
+	 * 	Document Status is Complete, Closed, Reversed or Voided
+	 *	@return true if CO, CL, RE or VO
 	 */
 	public boolean isComplete()
 	{
 		String ds = getDocStatus();
-		return DOCSTATUS_Completed.equals(ds) 
+		return DOCSTATUS_Completed.equals(ds)
 			|| DOCSTATUS_Closed.equals(ds)
-			|| DOCSTATUS_Reversed.equals(ds);
+			|| DOCSTATUS_Reversed.equals(ds)
+			|| DOCSTATUS_Voided.equals(ds);

```

Comment:
This is probably a change that might break some existing code/expectation.

Reply:
Found an example of that at InvoiceGenerate, line 267

Reply:
Changed back all the changes to isComplete to let them as before, and change the logics in beforeSave to use isProcessed instead of isComplete.

The example at InvoiceGenerate:267 looks like a bug, in theory harmless as is "processing" void shipments - but in theory void shipments must have quantities in zero.